pandas/plotting/_core.py

```diff
@@ -2061,7 +2061,7 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
         For data grouped with ``by``:
 
         * :class:`~pandas.Series`
-        * :class:`~numpy.array` (for ``return_type = None``)
+        * :class:`~numpy.array` (for ``return_type = None``).
```

Comment:
Similar comment here. Maybe move the majority of this into Notes and have the description here just say "See Notes."

Reply:
agreed, I moved the majority of the return documentation into notes. Let me know what you think.

One alternative would be to simply say 'See Notes.' instead of including the return type.

Reply:
Same comment